packages/eslint-plugin/tests/rules/array-type.test.ts

```diff
@@ -254,18 +254,18 @@ ruleTester.run('array-type', rule, {
     },
     {
       code: `
-function fooFunction(foo: Array<ArrayClass<string>>) {
-  return foo.map(e => e.foo);
-}
-      `,
```

Comment:
[Style] Sorry if I missed this before - could you please revert any unrelated changes, such as differences in indentation levels? It's harder to review the PR when there's a lot of irrelevant noise in the diff.